src/Sylius/Bundle/ApiBundle/Command/SendContactRequest.php

```diff
@@ -16,13 +16,11 @@
 /** experimental */
 class SendContactRequest implements ChannelCodeAwareInterface, LocaleCodeAwareInterface, LoggedInCustomerEmailIfNotSetAwareInterface
 {
-    public ?string $localeCode = null;
-
-    public ?string $channelCode = null;
-
     public function __construct(
-        private ?string $email = null,
-        private ?string $message = null,
+        protected ?string $channelCode,
+        protected ?string $localeCode,
+        protected ?string $email = null,
```

Comment:
Is default `null` value necessary for something or it might be left over? Other classes doesn't have default values.